homeassistant/components/spc/config_flow.py

```diff
@@ -0,0 +1,95 @@
+"""Config flow for SPC integration."""
+import logging
+
+from pyspcwebgw import SpcWebGateway
+import voluptuous as vol
+
+from homeassistant import exceptions
+from homeassistant.config_entries import CONN_CLASS_LOCAL_PUSH, ConfigFlow
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import aiohttp_client
+
+from . import CONF_API_URL, CONF_WS_URL, DOMAIN  # pylint: disable=unused-import
+
+_LOGGER = logging.getLogger(__name__)
+
+DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(
+            CONF_API_URL, description={"suggested_value": "http://<IP>:8088"}
+        ): str,
+        vol.Required(
+            CONF_WS_URL, description={"suggested_value": "ws://<IP>:8088/ws/spc"}
+        ): str,
```

Comment:
Wouldn't it be better to split this up to host address and port? Both should be the same per device right?
```suggestion
        vol.Required(CONF_HOST): str,
        vol.Required(CONF_PORT, default=8088): int,
```

Reply:
It's *probably* the same in almost all cases. However, there may be scheme variations (http/https, ws/wss) and who knows if someone has put the device behind a reverse proxy or such. Since the flexibility is already there in the existing versions I'd say it's better to continue to leave it open, at least for now.

Reply:
You could look at offering a simple flow doing that, and an advanced way to offer full configuration. I just wanted to point it out. Core dev might pitch in here